src/js/vic-v2/components/CropperController.jsx

```diff
@@ -0,0 +1,480 @@
+    this.refs.cropper.setCanvasData(position);
+  }
+
+  // explicit positioning
+  movePhotoToDefaultPosition = () => {
+    const defaultPhotoPosition = getDefaultPhotoPosition({
+      photoData: this.refs.cropper.getCanvasData(),
+      cropBoxData: this.refs.cropper.getCropBoxData(),
+      containerWidth: this.refs.cropper.getContainerData().width
+    });
+    this.movePhotoToPosition(defaultPhotoPosition);
+  }
+
+  // moves photo within bounds
+  movePhoto = ({ x = 0, y = 0 } = {}) => {
+    const photoData = this.refs.cropper.getCanvasData();
+    const cropBoxData = this.refs.cropper.getCropBoxData();
+    const boundaries = getPhotoBoundaries({ photoData, cropBoxData });
+
+    const newPhotoPosition = getNewPhotoPosition({ photoData, boundaries, x, y });
+
+    this.movePhotoToPosition(newPhotoPosition);
+  }
+
+  // relative positioning
+  rotatePhoto = (degrees) => {
+    this.refs.cropper.rotate(degrees);
+  }
+
+  zoomPhoto = (direction) => {
+    switch (direction) {
+      case 'IN':
+        if (this.state.zoomValue < this.state.zoomMax) {
+          this.refs.cropper.zoom(0.1);
+        }
+        break;
+      case 'OUT':
+        if (this.state.zoomValue > this.state.zoomMin) {
+          this.refs.cropper.zoom(-0.1);
+        }
+        break;
+      default:
+    }
+  }
+
+  updateWarningAndButtonState = () => {
+    // update warnings and buttons enabled state
+    const boundariesMet = this.getBoundariesMet();
+    const warningMessage = getWarningMessage(boundariesMet);
+    const buttonsDisabled = getMoveButtonsDisabled(boundariesMet);
+
+    this.setState({
+      warningMessage,
+      ...buttonsDisabled
+    });
+  }
+
+  updateZoomToDefaultState = () => {
+    const photoData = this.refs.cropper.getCanvasData();
+    const zoomMin = photoData.width / photoData.naturalWidth;
+
+    this.setState({
+      zoomMin,
+      zoomValue: zoomMin,
+    });
+  }
+
+  render() {
+    return (
+      <div className="cropper-container-outer">
+        <Cropper
+          ref="cropper"
+          key={this.props.narrowLayout ? 'narrowLayout' : 'normalLayout'}
+          ready={this.setCropBox}
+          responsive
+          src={this.props.src}
+          aspectRatio={1}
+          cropBoxMovable={false}
+          cropstart={this.onCropstart}
+          cropend={this.onCropend}
+          cropmove={() => true}
+          minContainerHeight={this.props.narrowLayout ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE}
+          toggleDragModeOnDblclick={false}
+          dragMode="move"
+          guides={false}
+          viewMode={0}
+          zoom={this.onZoom}/>
+        <div className="cropper-zoom-container">
+          <button className="cropper-control cropper-control-zoom cropper-control-zoom-in va-button va-button-link" type="button" onClick={this.onZoomOut}>
+            <span className="cropper-control-label">{this.props.narrowLayout || 'Make smaller'}<i className="fa fa-search-minus"></i></span>
```

Comment:
This icon will always have a span around it now but it seems to work the same and made the code [simpler](https://github.com/department-of-veterans-affairs/vets-website/blob/5d6dc7f6ec81286911ace84f4a72c7c9ba78e3a1/src/js/vic-v2/components/PhotoField.jsx#L744).